amnestywebsite/humanity-theme
# Eurasia site - change translation on sort by filter

Changes the Most Recent and Oldest First translations for post filter…s on eurasia site

Epic: https://github.com/amnestywebsite/humanity-theme/issues/417
Issue: https://github.com/amnestywebsite/amnesty-wp-theme/issues/2938

**Steps to test**:
1. Navigate to the search page on a russian language site
2. Use the sort by dropdown and check Most Recent and Oldest First translations match those in the ticket open

**Considerations**:
- Localisation
